docs/changelog/0.4.11.md

```diff
@@ -0,0 +1,178 @@
+
+# 0.4.11 (2023-03-15)
+
+We're happy to announce the release of Scala Native. It's the next maintenance release for Scala Native 0.4.x. As always it brings bug fixes and minor improvements.
+
+## Notable changes
+
+### Extern methods with a variadic number of arguments
+For a long time, Scala Native supported C `va_list` using `scalanative.unsafe.CVarArgList`. This allowed for interop with some of the C functions taking the variadic number of arguments. This release makes usage and definition of them easier, by restoring support for idiomatic ways of passing them using Scala variadic arguments lists.
+
+### Support for LLVM 15
+The latest versions of LLVM added a new internal representation of pointers - their opaque variant replaces typed pointers. This change should not affect the used of the LLVM toolchain, but in some specific builds it could have lead to linking issues.
```

Comment:
Noticed when reading at a high level.  Since these notes
are draft, thought I would point it out.

> change should not affect the used of the
--------------------------------------------???????

Overall, the release notes are quite nice, especially the varargs discussion.